### PR TITLE
MH3: Emrakul, the World Anew + 2 cards

### PR DESCRIPTION
**Pull request by MikeS-NZ**
MH3: Add card scripts for three MH3 cards:

- Emrakul, the World Anew
- Grist, Voracious Larva // Grist, the Plague Swarm
- Phlage, Titan of Fire's Fury

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/grist_voracious_larva_grist_the_plague_swarm.txt [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+SVar:DBMill:DB$ Mill | Defined$ You | NumCards$ 2 | RememberMilled$ True | SubAbility$ DBPutCounter
+SVar:DBPutCounter:DB$ PutCounter | Defined$ Imprinted | CounterType$ Deathtouch | CounterNum$ 1 | ConditionDefined$ Remembered | ConditionPresent$ Card.Black | ConditionCompare$ EQ1 | SubAbility$ DBCleanup
+A:AB$ Destroy | Cost$ SubCounter<2/LOYALTY> | Planeswalker$ True | ValidTgts$ Artifact,Enchantment | TgtPrompt$ Select target artifact or enchantment | SpellDescription$ Destroy target artifact or enchantment.
+A:AB$ RepeatEach | Cost$ SubCounter<6/LOYALTY> | Planeswalker$ True | Ultimate$ True | RepeatCards$ Card.Creature+YouOwn | Zone$ Graveyard | RepeatSubAbility$ DBCopy | SpellDescription$ For each creature card in your graveyard, create a token that's a copy of it, except it's a 1/1 black and green Insect.
+SVar:DBCopy:DB$ CopyPermanent | Defined$ Remembered | SetPower$ 1 | SetToughness$ 1 | SetColor$ Black & Green | SetCreatureTypes$ Insect | AILogic$ DuplicatePerms | SubAbility$ DBCleanup
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/grist_voracious_larva_grist_the_plague_swarm.txt:26]: shouldn't need RepeatEach if you use `Defined$ ValidGraveyard`

File: forge-gui/res/cardsfolder/upcoming/grist_voracious_larva_grist_the_plague_swarm.txt [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+K:Deathtouch
+T:Mode$ ChangesZone | Origin$ Graveyard | Destination$ Battlefield | ValidCard$ Card.Self,Creature.YouCtrl | Execute$ TrigTransform | TriggerDescription$ When CARDNAME or another creature enters the battlefield under your control, if it entered from your graveyard or you cast it from your graveyard, you may pay {G}. If you do, exile NICKNAME, then return it to the battlefield transformed under its owner's control.
+T:Mode$ ChangesZone | Destination$ Battlefield | ValidCard$ Card.Self+wasCastFromYourGraveyardByYou,Creature.YouCtrl+wasCastFromYourGraveyardByYou | Execute$ TrigTransform | Secondary$ True | TriggerDescription$ When CARDNAME enters the battlefield, if it entered from your graveyard or you cast it from your graveyard, you may pay {G}. If you do, exile NICKNAME, then return it to the battlefield transformed under its owner's control.
+SVar:TrigTransform:AB$ ChangeZone | Cost$ G | Origin$ Battlefield | Destination$ Exile | RememberChanged$ True | SubAbility$ DBReturn
+SVar:DBReturn:DB$ ChangeZone | Defined$ Remembered | Origin$ Exile | Destination$ Battlefield | Transformed$ True | ForgetOtherRemembered$ True | SubAbility$ DBCleanup | StackDescription$ None
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/grist_voracious_larva_grist_the_plague_swarm.txt:9]: stackdescription?

File: forge-gui/res/cardsfolder/upcoming/grist_voracious_larva_grist_the_plague_swarm.txt (hunk 1 above)
Comment (MikeS-NZ) [reply forge-gui/res/cardsfolder/upcoming/grist_voracious_larva_grist_the_plague_swarm.txt]: Reworked it using `Defined$ ValidGraveyard`. Thanks!

File: forge-gui/res/cardsfolder/upcoming/grist_voracious_larva_grist_the_plague_swarm.txt (hunk 2 above)
Comment (MikeS-NZ) [reply forge-gui/res/cardsfolder/upcoming/grist_voracious_larva_grist_the_plague_swarm.txt]: Removed!

**Review by tool4ever — APPROVED** [review]

**Pull request merged**